load_testing/lib/flow_ial2_proofing.py

```diff
@@ -44,110 +49,197 @@ def do_ial2_proofing(context):
     resp = do_request(
         context,
         "put",
-        "/verify/doc_auth/upload?type=desktop",
+        "/verify/hybrid_handoff",
         "/verify/document_capture",
         "",
         {"authenticity_token": auth_token, },
     )
-    auth_token = authenticity_token(resp)
+    dom = resp_to_dom(resp)
+    selector = 'meta[name="csrf-token"]'
+    auth_token = dom.find(selector).eq(0).attr("content")
+
+    selector = 'input[id="doc_auth_document_capture_session_uuid"]'
+    dcs_uuid = dom.find(selector).eq(0).attr("value")
 
-    files = {"doc_auth[front_image]": context.license_front,
-             "doc_auth[back_image]": context.license_back}
+    second_auth_token = authenticity_token(resp)
 
+    files = {"front": context.license_front,
+             "back": context.license_back,
+             }
+
+    if os.getenv("DEBUG"):
+        print("DEBUG: /api/verify/images")
     # Post the license images
+    resp = do_request(
+        context,
+        "post",
+        "/api/verify/images",
+        None,
+        None,
+        {
+            "flow_path": "standard",
+            "document_capture_session_uuid": dcs_uuid},
+        files,
+        None,
+        {"X-CSRF-Token": auth_token},
+    )
+
+    if os.getenv("DEBUG"):
+        print("DEBUG: /verify/document_capture")
     resp = do_request(
         context,
         "put",
         "/verify/document_capture",
-        "/verify/doc_auth/ssn",
-        "",
-        {"authenticity_token": auth_token, },
-        files
+        "/verify/ssn",
+        None,
+        {
+            "_method": "patch",
+            "doc_auth[document_capture_session_uuid]": dcs_uuid,
+            "authenticity_token": second_auth_token,
+        },
     )
     auth_token = authenticity_token(resp)
 
-    ssn = '900-12-3456'
-    # print("*** Using ssn: " + ssn)
+    ssn = f'900-12-{random.randint(0,9999):04}'
```

Comment:
how the change to be random is not required, right?

Reply:
I don't think so. @mitchellhenke, do you recall why we added that?

Reply:
I remember something coming up, but I don't recall exactly